platform/mikro-e/net-init.c

```diff
@@ -72,7 +112,7 @@ net_init()
   #ifdef __USE_CC2520__
     cc2520_get_random((uint8_t *)&longaddr, sizeof(longaddr));
   #elif  __USE_CA8210__
-    ca8210_get_random((uint8_t *)&longaddr, sizeof(longaddr));
+    longaddr = get_mac_address();
```

Comment:
The TI chips schematics show that they don't have the NVS module, so to maintain backwards compatibility we have to use the random function for cc2520